code/game/objects/items/weapons/power_cells.dm

```diff
@@ -29,6 +29,10 @@
 	origin_tech = list(TECH_POWER = 6)
 	maxcharge = 20000
 
+/obj/item/cell/large/hyper/depleted
+	charge = 0
```

Comment:
`charge` is set to `0` by default. all you need is the `starts_max_charge` below.

Reply:
i forgot i added this, i dont even need it anymore